### PR TITLE
Created contrast function

### PR DESCRIPTION
**Pull request by GeorgeJJW**
Added:
- Implemented contrast function with exceptions handling, all related tests to this function should pass.

Changes:
- Modified test suites, and fixed image paths.
- Adjusted file structure so that pytest is working properly.
- Cleaned up unnecessary files in the repository.

### CODE REVIEW

File: picfixPy/contrast.py [hunk 1]
```diff
@@ -16,5 +22,80 @@ def contrast(input_img, intensity=5, output_img):
...
     ============
-    contrast("../img/image.png", 2, "../img/contrast_image.png")
-    '''
\ No newline at end of file
+    contrast("../img/image.png", 5, display=False, "../img/contrast_image.png")
+    '''
+
+    # load input image
+    # lift pixel value restrictions between 0 and 255 temporarily
+    try:
```
Comment (mkeyim) [inline picfixPy/contrast.py:29]: I like the exception handling in the function, I'll add this to my function too

File: picfixPy/contrast.py [hunk 2]
```diff
@@ -16,5 +22,80 @@ def contrast(input_img, intensity=5, output_img):
...
+        raise
+    
+    except OSError:
+        print("Please provide the path to an image file.")
+        raise
+    
+    except FileExistsError:
+        print("Cannot find image file.")
+            
+    except Exception as error:
```
Comment (mkeyim) [inline picfixPy/contrast.py:43]: should we include an exception for image type?

File: picfixPy/contrast.py [hunk 3]
```diff
@@ -16,5 +22,80 @@ def contrast(input_img, intensity=5, output_img):
...
+        print("Cannot load image, something went wrong :(")
+        print(error)
+        raise
+
+    # validate intensity level
+    if (intensity < 0 or intensity > 10):
```
Comment (mkeyim) [inline picfixPy/contrast.py:49]: This is a good point, I'll add in the same into mine.

File: picfixPy/contrast.py [hunk 4]
```diff
@@ -16,5 +22,80 @@ def contrast(input_img, intensity=5, output_img):
...
+            return x
+    
+    np_restrict = np.vectorize(restrict)
+    enhanced_img = np_restrict(enhanced_img).astype(int)
+
+    # display or save enhanced image
+    if (display == True):
+        skimage.io.imshow(enhanced_img)
+    
+    if (output_img != ''):
```
Comment (mkeyim) [inline picfixPy/contrast.py:82]: This is great, I'll add in my function as well!

File: picfixPy/test/test_sharpen.py [hunk 5]
```diff
@@ -9,15 +9,15 @@
...
+from picfixPy.sharpen import sharpen
 
 # generate input image
 
 test_img1 = np.array([[[ 10,  20,  40], [ 20,  40,  10], [ 40,  10,  20]],
                       [[ 40,  80, 160], [ 80, 160,  40], [160,  40,  80]],
                       [[ 60, 120, 240], [120, 240,  60], [240,  60, 120]]], dtype = "uint8")
 
-skimage.io.imsave("pixfixPy/test/test_img/sharpen/test_img1.png", test_img1)
+skimage.io.imsave("picfixPy/test/test_img/sharpen/test_img1.png", test_img1)
```
Comment (mkeyim) [inline picfixPy/test/test_sharpen.py:20]: good catch, I'll try to sift through my code as well for pesty spelling errors.

File: picfixPy/test/test_vibrance.py [hunk 6]
```diff
@@ -9,15 +9,15 @@
 import numpy as np
 import pytest
 import skimage.io
-from picfixPy import vibrance
+from picfixPy.vibrance import vibrance
```
Comment (mkeyim) [inline picfixPy/test/test_vibrance.py:12]: thanks for the update! I'll continue the test once this has been merged.

Comment (GeorgeJJW) [reply picfixPy/test/test_vibrance.py:12]: A pleasure!

File: picfixPy/contrast.py (hunk 2 above)
Comment (GeorgeJJW) [reply picfixPy/contrast.py:43]: Nice catch!
The OSError is meant for catching illegal file type, so I rewrote the error message to be more clear!

The new error message as of https://github.com/UBC-MDS/picfixPy/pull/17/commits/86a2f2ca301268f11318b5c5cf7ad24f3f7f31af now reads:
```
    except OSError:
        print("Please provide an image file.")
        raise
```

**Comment by GeorgeJJW** [comment]
Thanks for the feedback!

**Pull request merged**